pkg/integrations/v4/supervisor_fb.go

```diff
@@ -31,6 +36,33 @@ type FBSupervisorConfig struct {
 	FluentBitVerbose     bool
 }
 
+const (
+	MaxNumberOfFbConfigTempFiles int = 50
+)
+
+// ErrorsList error representing a list of errors
+type ErrorsList struct {
```

Comment:
nice! I would not export ☝️  it though :)

Reply:
I would change it to:
```
type errorsList struct {
```

as this structure is not meant to be used outside :)

Reply:
Ah, ok! I'll do it!